README.md

```diff
@@ -1,9 +1,102 @@
 [![Build Status](https://travis-ci.org/sogilis/csv_fast_importer.svg?branch=master)](https://travis-ci.org/sogilis/csv_fast_importer)
 
-# csv_fast_importer
-Fast CSV Importer which use COPY PostgreSQL command
+# CSV Fast Importer
 
-Dev in progress...
+A gem to import CSV files' content into a PostgreSQL database. It is based on
+the [Postgre `COPY` command](https://wiki.postgresql.org/wiki/COPY) which is
+not standard but faster than other solutions.
```

Comment:
More than "faster than other solution", `COPY` is designed to be the fastest solution to import data in PostgreSQL database.